openach/views/boards.py

```diff
@@ -50,11 +52,18 @@
 @account_required
 def board_listing(request):
     """Return a paginated board listing view showing all boards and their popularity."""
-    board_list = Board.objects.user_readable(request.user).order_by("-pub_date")
+    board_obj = Board.objects.user_readable(request.user).annotate(
+        board_title_lower=Lower("board_title")
+    )
+    board_list = custom_sort(request, board_obj)
     metric_timeout_seconds = 60 * 2
     desc = _("List of intelligence boards on {name} and summary information").format(
         name=get_current_site(request).name
     )  # nopep8
+    name = request.GET.get("sort")
```

Comment:
A better name for this might be `sort_field`. `name` is confusing because it sounds like it might refer to the board name